src/Microsoft.Framework.PackageManager/Bundle/BundleManager.cs

```diff
@@ -284,5 +289,49 @@ private DependencyContext CreateDependencyContext(Runtime.Project project, Frame
             dependencyContext.Walk(project.Name, project.Version);
             return dependencyContext;
         }
+
+        private bool ResolveActualRuntimeNames(IList<string> runtimes)
+        {
+            for (int i = 0; i < runtimes.Count(); i++)
+            {
+                if (string.Equals("active", runtimes[i], StringComparison.OrdinalIgnoreCase))
+                {
+                    string activeRuntime = GetActiveRuntimeName();
+                    if (string.IsNullOrEmpty(activeRuntime))
+                    {
+                        _options.Reports.Error.WriteLine("Cannot resolve the active runtime name".Red());
+                        return false;
+                    }
+
+                    _options.Reports.Verbose.WriteLine("Resolved the active runtime as {0}", activeRuntime);
+                    runtimes[i] = activeRuntime;
+                }
+            }
+
+            return true;
+        }
+
+        private static string GetActiveRuntimeName()
```

Comment:
Interesting to see this logic. Look like very general, can we pull it in a helper class?